test/ctia/test_helpers/http.clj

```diff
@@ -42,7 +42,9 @@
               :body new-entity
               :headers {"api_key" api-key})]
     (when (not= 200 status)
-      (throw (ex-info (str "Expected status to be 200 but was %s" status)
+      (throw (ex-info (str "Expected status to be 200 but was " status
+                           " for " path ":\n"
+                           (with-out-str (clojure.pprint/pprint response)))
```

Comment:
The ex-info fn takes a map as second arg.  The response should be that map.  If you want to print out errors like this, add a (try ... (catch ...)) to ctia.init somewhere that uses ex-data to get the map and pprint it.  Its more idiomatic and lets you avoid having a lot of this kind of (with-out-str ...) code all over.